.travis.yml

```diff
@@ -60,5 +61,6 @@ script:
     # - go run github.com/myitcv/gobin -m -run myitcv.io/cmd/mdreplace -long -online -w README.md
     # - gobin -m -run myitcv.io/cmd/mdreplace -w README.md
 
+  - ./_scripts/apply_known_diffs.sh
```

Comment:
Go 1.13 fixed the sorting of `vendor/modules.txt`. However, this breaks out check on prior versions of Go. Hence, apply known diffs to satisfy the git status check.

Reply:
Maybe add this as a comment?

Reply:
Good idea. Done